adam-core/src/main/scala/org/bdgenomics/adam/converters/AlignmentRecordConverter.scala

```diff
@@ -42,13 +47,8 @@ class AlignmentRecordConverter extends Serializable {
     val readNameSuffix =
       if (maybeAddSuffix &&
         !AlignmentRecordConverter.readNameHasPairedSuffix(adamRecord) &&
-        adamRecord.getFirstOfPair != null) {
-        "/" + (
-          if (adamRecord.getFirstOfPair)
-            "1"
-          else
-            "2"
-        )
+        adamRecord.getReadPaired) {
+        "/%d".format(adamRecord.getReadNum + 1)
       } else {
         ""
       }
```

Comment:
Just below here, when we create the qualityScores string, e.g.

```
val qualityScores =
       if (outputOriginalBaseQualities && adamRecord.getOrigQual != null)
         adamRecord.getOrigQual
       else
         adamRecord.getQual
```

We need to update this to be changed I think. If a user requests the original quality scores, and they don't exist, they will be get the updated scores without as much as an error.

Something like, e.g.

```
val qualityScores =
      if (outputOriginalBaseQualities)
        Option(adamRecord.getOrigQual).getOrElse("*")
      else
        adamRecord.getQual
```

would be better.